src/deploy/D3MInit.sol

```diff
@@ -113,19 +119,25 @@ struct D3MAaveConfig {
     uint256 adaiRevision;
 }
 
-struct D3MCompoundConfig {
+struct D3MCompoundPoolConfig {
     address king;
-    uint256 barb;
     address cdai;
     address comptroller;
     address comp;
+}
+
+struct D3MCompoundPlanConfig {
+    uint256 barb;
+    address cdai;
     address tack;
     address delegate;
 }
 
 // Init a D3M instance
 library D3MInit {
 
+    using ScriptTools for string;
```

Comment:
Same here, not sure where we need that strings inherit the ScriptTools properties.